### PR TITLE
add two features in soc

### PR DESCRIPTION
**Pull request by GregoryIan**
@shenli PTAL

### CODE REVIEW

File: soc/README.md [hunk 1]
```diff
@@ -93,4 +93,42 @@ TiDB(localhost:4000) > desc select max(b), sum(c) from t group by a;
...
+project example: [TiDB-operator](https://github.com/pingcap/tidb-operator)
+
+-	Recommended Skills: golang
+-	Issue: https://github.com/pingcap/dm/issues/43
+
+### **visual display of DM**
+
+-	Description
+
+Now what information does dm have to show to users?
```
Comment (lilin90) [inline soc/README.md]:
```suggestion
Now what information does DM have to show to users?
```

File: soc/README.md [hunk 2]
```diff
@@ -93,4 +93,42 @@ TiDB(localhost:4000) > desc select max(b), sum(c) from t group by a;
...
+
+* dmctl
+
+  * `query-status`: query basic information of task, contain some complex and not clear error message
```
Comment (lilin90) [inline soc/README.md]:
```suggestion
  * `query-status`: queries basic information of the task, including some complex and unclear error messages
```

File: soc/README.md [hunk 3]
```diff
@@ -93,4 +93,42 @@ TiDB(localhost:4000) > desc select max(b), sum(c) from t group by a;
...
+  * `show-ddl-lock` ....
+  * `query-error` ...
+* grafana - many no sense monitor graph
```
Comment (lilin90) [inline soc/README.md]:
```suggestion
* Grafana: many no sense monitor graph
```

Comment (lilin90) [reply soc/README.md]: This line is confusing. Could you make it clearer?

File: soc/README.md [hunk 4]
```diff
@@ -93,4 +93,42 @@ TiDB(localhost:4000) > desc select max(b), sum(c) from t group by a;
...
+
+What are the disadvantages of the above methods? **Lack of contextual information leads to incomprehensible or inferential problems**
```
Comment (lilin90) [inline soc/README.md]:
```suggestion
What are the disadvantages of the above methods? **Lack of contextual information leads to incomprehensible or inferential problems.**
```

File: soc/README.md [hunk 5]
```diff
@@ -93,4 +93,42 @@ TiDB(localhost:4000) > desc select max(b), sum(c) from t group by a;
...
+
+we need a way to show system or task running status details in natural way, like a straightforward way to show the speed of data flow, key events and where to happen
```
Comment (lilin90) [inline soc/README.md]:
```suggestion
We need a way to show the system or task running status details in a natural way, like a straightforward way to show the speed of data flow, key events and where to happen.
```

File: soc/README.md [hunk 6]
```diff
@@ -93,4 +93,42 @@ TiDB(localhost:4000) > desc select max(b), sum(c) from t group by a;
...
+
+-	Recommended Skills: golang, opentracing
```
Comment (lilin90) [inline soc/README.md]:
```suggestion
-	Recommended Skills: Golang, OpenTracing
```

**Comment by lilin90** [comment]
For each "Description" section, please add an indentation, which would make the content logic clearer. Ref: #22

File: soc/README.md [hunk 7]
```diff
@@ -93,4 +93,42 @@ TiDB(localhost:4000) > desc select max(b), sum(c) from t group by a;
...
+-	Description
+
+Now what information does DM have to show to users?
+
+* dmctl
+
+  * `query-status`: queries basic information of the task, including some complex and unclear error messages
+  * `show-ddl-lock` ....
+  * `query-error` ...
+* Grafana: many no sense monitor graph
```
Comment (lilin90) [inline soc/README.md]:
```suggestion
* Grafana: many nonsense monitor graphs
```

**Comment by GregoryIan** [comment]
thanks @lilin90  we can add indentation in #22

**Pull request merged**